### ISSUE TITLE
[Istio Prelim 1.0] Service Graph shows nothing

### ISSUE THREAD

**Issue by igoooor** [istio/istio#7414]
**Describe the bug**
Service Graph returns empty data `{"nodes":[],"links":[]}`

**Expected behavior**
I expected to see my services from the bookinfo example

**Steps to reproduce the bug**
- install Istio 1.0.0-snapshot.2 with helm
`helm install ./istio-1.0.0-snapshot.2/install/kubernetes/helm/istio --name istio --namespace istio-system  --set servicegraph.enabled=true`
- add auto injection
`kubectl label namespace default istio-injection=enabled`
- install bookinfo project
`kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml`
`istioctl create -f samples/bookinfo/networking/bookinfo-gateway.yaml`
- confirm the app is running (and to send traffic to the mesh.)
`curl -o /dev/null -s -w "%{http_code}\n" http://${GATEWAY_URL}/productpage`
- Verify that the service graph is running in your cluster.
`kubectl -n istio-system get svc servicegraph`
- Open the Servicegraph UI
`kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=servicegraph -o jsonpath='{.items[0].metadata.name}') 8088:8088 &`
`Visit http://localhost:8088/d3graph`

Please note that following the exact same steps but with Istio 0.8 perfectly works.
I'm not an expert, but I'll be happy to follow any steps necessary to help with debugging

**Version**
```
Version: 1.0.0-snapshot.2
GitRevision: a66a221bfe4cc068abad0c1fba93093600a5c6ef
User: root@661f2acc021d
Hub: gcr.io/istio-release
GolangVersion: go1.10.1
BuildStatus: Clean
```
```
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T22:29:25Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"9+", GitVersion:"v1.9.7-gke.4", GitCommit:"9b635efce81582e1da13b35a7aa539c0ccb32987", GitTreeState:"clean", BuildDate:"2018-06-28T23:30:47Z", GoVersion:"go1.9.3b4", Compiler:"gc", Platform:"linux/amd64"}
```

**Environment**
GCP

**Comment by munnerz** [comment]
I've opened #7378 which should resolve this issue

**Comment by munnerz** [comment]
This can now be closed as it is fixed with #7421 

/close
